source/transactions/tests/write-concern.yml

```diff
@@ -400,10 +389,7 @@ tests:
           update:
             $inc: {x: 1}
           upsert: true
-        result:
-          matchedCount: 1
-          modifiedCount: 1
-          upsertedCount: 0
+        # avoid assertion since result fields are inaccessible for unacknowledged write concern
```

Comment:
This command is part of a transaction so the driver ignores the w:0 write concern. A result *is* returned.